controllers/issue.js

```diff
@@ -260,8 +260,8 @@ exports.view = function (aReq, aRes, aNext) {
       }
 
       // Discussion
-      discussion = options.discussion = modelParser.parseDiscussion(aDiscussion);
-      modelParser.parseIssue(discussion);
+      options.discussion = discussion = modelParser.parseDiscussion(aDiscussion);
+      modelParser.parseIssue(discussion); // WARNING: Return value never used
```

Comment:
Alright... just making sure we're on the same page... I know I use a little different nomenclature from testing aspect and my software engineering history. Thank you for the clarification.